core/src/main/resources/error/error-classes.json

```diff
@@ -870,12 +876,24 @@
     ],
     "sqlState" : "42000"
   },
+  "INVALID_DRIVER_MEMORY" : {
+    "message" : [
+      "System memory <systemMemory> must be at least <minSystemMemory>. Please increase heap size using the --driver-memory option or \"<config>\" in Spark configuration."
+    ],
+    "sqlState" : "F0000"
```

Comment:
@srielau, could you suggest on the sqlState here? It seems that we don't have one for invalid config?

Reply:
Yep, we don't have. Let's assign this one.